build.sbt

```diff
@@ -389,8 +389,8 @@ lazy val docs = project
 
 lazy val cats = project
   .in(file("."))
-  .settings(moduleName := "root")
-  .settings(catsSettings)
+  .settings(moduleName := "root", crossScalaVersions := Nil)
+  .settings(publishSettings)
```

Comment:
It looks a little weird to have `publishSettings` followed by `noPublishSettings`. This is intentional, right? If so, it might be helpful to have a brief comment about why this is being done.

Reply:
Good catch. I added some comment here. The basic thing is that `sbt release` command is run through aggregation from this root module. Thus it requires some release settings, then the `noPublishSettings` makes sure the module itself is not published.